modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapAnchor.vue

```diff
@@ -8,6 +8,11 @@
       :icon-only="!!tool.icon"
       :icon="tool.icon || false"
       :modifiers="['no-border', 'no-motion']"
+      :tooltip="{
+        content: tool.label,
+        placement: 'top',
+        delay: 650
```

Comment:
big delay on these because they should be obvious, but still present if they need them.